opendevin/core/config.py

```diff
@@ -162,8 +164,8 @@ class AppConfig(metaclass=Singleton):
         file_uploads_allowed_extensions: List of allowed file extensions for uploads. ['.*'] means all extensions are allowed.
     """
 
-    llm: LLMConfig = field(default_factory=LLMConfig)
-    agent: AgentConfig = field(default_factory=AgentConfig)
+    llms: Dict[str, LLMConfig] = {}
+    agents: Dict[str, AgentConfig] = {}
```

Comment:
```suggestion
    llms: dict[str, LLMConfig] = {}
    agents: dict[str, AgentConfig] = {}
```

Reply:
lol you are too fast. Hold on, not ready for review yet 🤣